server/views/pages/teamOverview.njk

```diff
@@ -4,263 +4,269 @@
+              <dt class="govuk-summary-list__key govuk-body govuk-!-font-size-48">
+                {{ team.veryHighAndHighVeracode }}
+              </dt>
+            </div>
+            <div class="govuk-summary-list__row">
+              <dd class="govuk-summary-list__value">
+                <a class="govuk-link" href="/veracode?team={{ team.encodedTeamName }}">See veracode scan results</a>
+              </dd>
+            </div>
+          </dl>
+        </div>
       </div>
     </div>
   </div>
-</div>
 
-{% if channelRecommendations %}
-<h3 class="govuk-heading-m">Monitoring Channel Recommendations</h3>
+  {% if team.channelRecommendations %}
+    <h3 class="govuk-heading-m">Monitoring Channel Recommendations</h3>
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
-    {% set hasLegacyChannels = false %}
-    {% set legacyChannelCount = 0 %}
-    {% for rec in channelRecommendations.recommendations %}
-      {% if rec.currentChannels.dev == '#dps_alerts_non_prod' or rec.currentChannels.preprod == '#dps_alerts_non_prod' or rec.currentChannels.prod == '#dps_alerts' %}
-        {% set hasLegacyChannels = true %}
-        {% set legacyChannelCount = legacyChannelCount + 1 %}
-      {% endif %}
-    {% endfor %}
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        {% set hasLegacyChannels = false %}
+        {% set legacyChannelCount = 0 %}
+        {% for rec in channelRecommendations.recommendations %}
+          {% if rec.currentChannels.dev == '#dps_alerts_non_prod' or rec.currentChannels.preprod == '#dps_alerts_non_prod' or rec.currentChannels.prod == '#dps_alerts' %}
+            {% set hasLegacyChannels = true %}
+            {% set legacyChannelCount = legacyChannelCount + 1 %}
+          {% endif %}
+        {% endfor %}
 
-    {% if hasLegacyChannels %}
-    <div class="govuk-warning-text" style="background-color: #fef7f7; padding: 20px; border-left: 5px solid #d4351c; margin-bottom: 30px; border-radius: 4px;">
-      <div style="display: flex; align-items: center; gap: 20px;">
-        <div style="background: linear-gradient(135deg, #d4351c 0%, #b91c1c 100%); color: white; border-radius: 50%; width: 50px; height: 50px; display: flex; align-items: center; justify-content: center; font-size: 20px; font-weight: bold; box-shadow: 0 2px 8px rgba(212, 53, 28, 0.3); flex-shrink: 0;">
-          {{ legacyChannelCount }}
-        </div>
-        <div style="flex: 1;">
-          <div style="display: flex; align-items: center; gap: 8px; margin-bottom: 5px;">
-            <span class="govuk-warning-text__icon" aria-hidden="true" style="font-size: 20px; color: #d4351c;">!</span>
-            <strong style="font-size: 16px; color: #d4351c;">Action Required</strong>
+        {% if hasLegacyChannels %}
+          <div class="govuk-warning-text" style="background-color: #fef7f7; padding: 20px; border-left: 5px solid #d4351c; margin-bottom: 30px; border-radius: 4px;">
```

Comment:
I know this is from previous PRs but this inline CSS needs stripping out and putting into classes in `local.scss`